### DISCUSSION TITLE
Can Autorest generate code that supports multiple versions?

### DISCUSSION THREAD

**Discussion by v-kaifazhang** [Azure/autorest#4428] [Q&A] [answered]
I want to use Autorest to generate Csharp code. Every time a feature is updated, the generated code only supports the current version of the function, so can Autorest support multiple versions of the function of the code?

**Comment by timotheeguerin** [comment, marked as answer]
There isn't a good way right now to have multiple version generated at once. You can use the `batch` config to do multiple client generation side by side.
